documentation/service_admin/assembly-installing-openshift.adoc

```diff
@@ -0,0 +1,30 @@
+// This assembly is included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+include::common/attributes.adoc[]
+
+[id='installing-on-openshift_{context}']
+= Installing {ProductName} on OpenShift
+
+{ProductName} can be installed using automated link:https://www.ansible.com/[Ansible] playbooks, the
+`deploy.sh` script, or the manual steps.
+
+*NOTE*: You can invoke the deployment script with `-h` to view a list of options.
+
+include::common/snip-prerequisites-openshift.adoc[]
+
+:parent-context: {context}
```

Comment:
Why is this here? I'd put it at the top right above the attributes.

Reply:
Fixed